docusaurus/docs/advanced-configuration.md

```diff
@@ -3,23 +3,21 @@ id: advanced-configuration
 title: Advanced Configuration
 ---
 
-The `.config/` directory holds the preferred configuration for the different tools
-that are used to develop, test and build the project.
+The `.config/` directory holds the preferred configuration for the different tools used to develop, test and build a plugin.
 
 :::danger
 
-In order to make updates easier do **not** edit files in the `.config` directory to extend configuration. Instead follow the directions in this article to customise the tooling configurations.
+To make future updates easier do **not** edit files in the `.config` directory. Instead follow the directions in this article to customise the tooling configurations.
 
 :::
 
-## How to extend the basic configs?
+## How to extend the configs?
 
-First note that you are doing this at your own risk and that extending any of the basic configuration can lead
-to issues around working with the project.
+Note that you are doing this at your own risk. Extending any of the basic configuration can lead to issues at build and runtime.
```

Comment:
Can we be any more descriptive with the sort of "issues"? e.g., "data loss issues" or "performance issues" or "site failure issues", etc.